products/claim-appeal-status/engineering/my-va-dashboard-discovery.md

```diff
@@ -0,0 +1,43 @@
+# Discovery: Claims and Appeals Access Issue on My VA Dashboard
+
+**Environment:** Staging  
+**Affected User:** Staging User 220  
+**Date:** 2025-10-16  
+
+## Summary
+A logic error in the My VA dashboard prevents users from accessing their claims information when the Appeals API returns an error, even when the Claims API is functioning correctly.
+
+## Current Behavior
```

Comment:
Could update this to be "Steps to Reproduce" as that is a bit more common and just make it a 5 item list that includes all the below steps (remove the 2 headings)